<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CityGMLModelGeneratorDialogBase</class>
 <widget class="QDialog" name="CityGMLModelGeneratorDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CityGMLモデル生成</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>ベースデータ読み込み</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,1">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_required">
          <property name="text">
           <string>必須項目</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_required_fields"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="button_basedata_folder">
          <property name="text">
           <string>フォルダ選択</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="basedata_folder_name">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_basedata_import">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>読み込み</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>属性情報読み込み</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5" columnstretch="1,1">
      <item row="5" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item row="0" column="2">
         <widget class="QComboBox" name="combo_csv_attribute_field">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_attribute_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>高さフィールド</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_attribute_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>リンクキー</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="combo_csv_link_key">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radio_button_feature">
        <property name="text">
         <string>地物データの属性を使用</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_attribute_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>高さフィールド</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_feature_attribute_field"/>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="button_attribute_setting">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>属性設定</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radio_button_csv">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>外部CSVの属性を使用</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="button_basedata_folder_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>ファイル選択</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="basedata_folder_name_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_attribute_import">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>読み込み</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_close">
       <property name="text">
        <string>閉じる</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>CityGML</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="button_citygml_export_folder">
          <property name="text">
           <string>出力フォルダ選択</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="citygml_export_folder_name"/>
        </item>
       </layout>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="button_citygml_export">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>出力</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>出力</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="button_citygml_check_output">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Debug 検査</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>button_basedata_folder</tabstop>
  <tabstop>basedata_folder_name</tabstop>
  <tabstop>button_basedata_import</tabstop>
  <tabstop>combo_required_fields</tabstop>
  <tabstop>button_basedata_folder_3</tabstop>
  <tabstop>basedata_folder_name_3</tabstop>
  <tabstop>button_attribute_import</tabstop>
  <tabstop>radio_button_feature</tabstop>
  <tabstop>combo_feature_attribute_field</tabstop>
  <tabstop>radio_button_csv</tabstop>
  <tabstop>combo_csv_attribute_field</tabstop>
  <tabstop>combo_csv_link_key</tabstop>
  <tabstop>button_attribute_setting</tabstop>
  <tabstop>button_citygml_export_folder</tabstop>
  <tabstop>citygml_export_folder_name</tabstop>
  <tabstop>button_citygml_export</tabstop>
  <tabstop>button_citygml_check_output</tabstop>
  <tabstop>button_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
